### ISSUE TITLE
Distillation from Transformer to CNN model

### ISSUE THREAD

**Issue by seanyeo300** [sunshangquan/logit-standardardization-KD#19]
Dear Author,

感谢您提供简单有效的方法来提高蒸馏性能。我正在尝试将这种方法用于不同架构之间的蒸馏。我的教师模型的logits来自一个transformer模型的集成，而我的学生模型是基于MobileNet的模型。在使用标准化时，我的准确率似乎降低了约2%。对于具有不同架构的学生和教师模型来说，这是预期的吗？比较是基于KD和KD+Ours

谢谢！

**Comment by sunshangquan** [comment]
Hi @seanyeo300 ，谢谢对我们工作的关注！在使用新的学生/教师模型设定的时候需要调整一下参数，直接不变的话可能会使准确率降低，参考[ViT logs](https://github.com/sunshangquan/logit-standardization-KD/tree/master/logs/tiny-transformer)里面的情况，beta和base-temp有可能需要改成3/6或者3/10之类的其他数值。

**Comment by seanyeo300** [comment]
谢谢您的快速回复！beta 只适用于 DKD，对吗？如果我只使用 KD，我只需要调整 base-temp？

**Comment by sunshangquan** [comment]
不好意思说错了，是kd-weight和base-temp，即--base-temp 和 --kd-weight

**Comment by seanyeo300** [comment]
非常感谢，我会尝试的

**Comment by seanyeo300** [comment]
抱歉重新打开这个问题，我有几种模型组合，想尝试你的方法。不过，我需要为每个模型调整 --base-temp 和 --kd-weight。对于设置 --base-temp 和 --kd-weight 的具体数值，有什么直观的建议吗？

还有一个问题，我们如何选择 --kd-weight 来影响训练？我只知道它决定了模型从 label loss 和 distill loss 中学习的程度。

感谢您的帮助。

**Comment by sunshangquan** [comment]
Hi @seanyeo300 , you could try several pairs of values for ```--base-temp```/```--kd-weight``` such as 2/3, 2/6, 2/9, 2/12, 2/15, 3/3, 3/6, 3/9, 3/12, 3/15 etc. By setting ```--kd-weight```, it balances the learning from hard labels of ground-truth and the learning from teacher's soft labels.

**Comment by sunshangquan** [comment]
Please feel free to reopen the issue for further question:)

**Comment by seanyeo300** [comment]
I see, thank you
